src/prismic.tests/FragmentsTests.cs

```diff
@@ -110,6 +110,21 @@ public void ShouldParseOEmbed()
 			Assert.IsNull (soundcloud.Obj.Width);
 			Assert.AreEqual (youtube.Obj.Width, 480);
 		}
+		
+		[Test()]
+		public void ShouldAccessRaw()
+		{
+		    var directory = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
```

Comment:
We already have an helper function for this: https://github.com/prismicio/csharp-kit/blob/master/src/prismic.tests/Fixtures.cs#L10. Could you use it?